docs/examples/viz_pbr_interactive.py

```diff
@@ -0,0 +1,337 @@
+"""
+===============================================
+Interactive PBR demo
+===============================================
+
+This is a demonstration of how Physically-Based Rendering (PBR) can be used to
+simulate different materials.
+
+Let's start by importing the necessary modules:
+"""
+
+from fury import actor, material, ui, window
+from fury.data import fetch_viz_cubemaps, read_viz_cubemap
+from fury.io import load_cubemap_texture
+from fury.utils import (normals_from_actor, tangents_to_actor,
+                        tangents_from_direction_of_anisotropy)
+
+
+"""
+The following functions will help us to manage the sliders events.
+"""
+
+
+def change_slice_metallic(slider):
+    global pbr_params, sphere
+    pbr_params['metallic'] = slider.value
+    sphere.GetProperty().SetMetallic(pbr_params['metallic'])
+
+
+def change_slice_roughness(slider):
+    global pbr_params, sphere
+    pbr_params['roughness'] = slider.value
+    sphere.GetProperty().SetRoughness(pbr_params['roughness'])
+
+
+def change_slice_anisotropy(slider):
+    global pbr_params, sphere
+    pbr_params['anisotropy'] = slider.value
+    sphere.GetProperty().SetAnisotropy(pbr_params['anisotropy'])
+
+
+def change_slice_anisotropy_direction_x(slider):
+    global doa, normals, sphere
+    doa[0] = slider.value
+    tangents = tangents_from_direction_of_anisotropy(normals, doa)
+    tangents_to_actor(sphere, tangents)
+
+
+def change_slice_anisotropy_direction_y(slider):
+    global doa, normals, sphere
+    doa[1] = slider.value
+    tangents = tangents_from_direction_of_anisotropy(normals, doa)
+    tangents_to_actor(sphere, tangents)
+
+
+def change_slice_anisotropy_direction_z(slider):
+    global doa, normals, sphere
+    doa[2] = slider.value
+    tangents = tangents_from_direction_of_anisotropy(normals, doa)
+    tangents_to_actor(sphere, tangents)
+
+
+def change_slice_anisotropy_rotation(slider):
+    global pbr_params, sphere
+    pbr_params['anisotropy_rotation'] = slider.value
+    sphere.GetProperty().SetAnisotropyRotation(
```

Comment:
@guaje For such functions we will need utility functions in material.py

Reply:
Changed. PTAL